docs/user/customization.md

```diff
@@ -64,4 +64,172 @@ An example `worker.ign` is shown below. It has been modified to increase the HTT
 }
 ```
 
+## Install Time Customization for Machine Configuration
```

Comment:
Should this go under [here][1], since that's the section about `manifests`?  Or do you want to avoid going into detail there and maybe just link out to this new section?

[1]: https://github.com/openshift/installer/blob/88d4374e0ae6c58c5cbea2431fe73a81d1334a06/docs/user/customization.md#kubernetes-customization-unvalidated